src/CustomFlags/CustomFlagsPlugin.cs

```diff
@@ -52,7 +52,7 @@ public override void OnInitialized()
 		}
 		
 		GameManager.Instance.Assets.LoadByLabel("custom_flag",
-			sprite => LoadCustomFlag(sprite, false), delegate(IList<Sprite> _)
+			sprite => LoadCustomFlag(sprite, true), delegate(IList<Sprite> _)
```

Comment:
Why is this now true? For images from disk I'm effectively registering and addressable at runtime, but this shouldn't be needed for flags that are already addressable

Reply:
Without this set to true I could pick an agency flag at main main, but when I entered the game and planted a flag, a regular/stock one would appear. I then experimented with changing this to true and it worked.